isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/nutzungsvorgaben/inhalt.adoc

```diff
@@ -372,6 +372,13 @@ public class OAuth2ServerSecurityConfig {
 }
 ----
 
+[INFO]
+====
+Bei der Verwendung von `isy-sicherheit` wurde der CSRF-Filter von Spring Security in vielen Fällen standarmäßig deaktiviert.
```

Comment:
Hier wird `isy-sicherheit` erwähnt, doch der Text steht im Baustein Security. Muss der Text an eine andere Stelle oder ist die erwähnte Bibliothek nicht korrekt?

Reply:
"isy-sicherheit" ist der Vorgänger von "isy-security". In "isy-sicherheit" ist das beschriebene Filter-Problem nicht aufgetreten und darauf soll an dieser Stelle in der "isy-security"-Dokumentation lediglich hingewiesen werden. Im späteren ergänzten Abschnitt (ab Zeile 378) wird dann beschrieben, wie die Filter-Problematik behoben werden kann, wenn der neue Baustein "isy-security" verwendet wird.